graphql-ballerina/validator_visitor.bal

```diff
@@ -21,14 +21,13 @@ class ValidatorVisitor {
 
     private __Schema schema;
     private parser:DocumentNode documentNode;
-    private int maxQueryDepth;
+    private int queryDepth = 0;
```

Comment:
Why do we need this variable?

Reply:
To keep query depth temporary.

Reply:
I don't see any usage of this variable here.

Reply:
This variable is not used. Shall we remove it?